start-symphony.ps1

```
@@ -0,0 +1,131 @@
+Write-Ok "  Linear:   $($linearKey.Substring(0,8))..."
+
+if ($DryRun) {
+    Write-Ok "Dry run complete. All checks passed."
+    exit 0
+}
+
+$env:PATH = "$OtpDir;$ElixirDir;$env:PATH"
+
+Set-Location $SymphonyDir
+
+$args = @(
```

PSScriptAnalyzer (1.25.0)

[warning] The Variable 'args' is an automatic variable that is built into PowerShell, assigning to it might have undesired side effects. If assignment is not by design, please use a different name.

(PSAvoidAssignmentToAutomaticVariable)